contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -5634,6 +5904,8 @@ bbf_ExecDropStmt(DropStmt *stmt)
 											 major_name, NULL);
 				}
 			}
+			if (logicalschema != NULL)
```

Comment:
When will logicalschema be NULL? Should we allow that?

Reply:
While dropping table valued function, when we drop the internal table then the logical schema becomes NULL.
We have to avoid that case.
Example from JDBC test framework:
```
CREATE FUNCTION my_splitstring_3101 ( @stringToSplit VARCHAR(MAX) )
RETURNS
@returnList TABLE ([Value] [nvarchar] (50))
AS
BEGIN
	DECLARE @name NVARCHAR(255)
	DECLARE @pos INT
	WHILE CHARINDEX(',', @stringToSplit) > 0
		BEGIN
			SELECT @pos  = CHARINDEX(',', @stringToSplit)
			SELECT @name = SUBSTRING(@stringToSplit, 1, @pos-1)
			INSERT INTO @returnList SELECT @name
			SELECT @stringToSplit = SUBSTRING(@stringToSplit, @pos+1, LEN(@stringToSplit)-@pos)
		END
		INSERT INTO @returnList SELECT @stringToSplit
		RETURN
END
GO
```
It executes `DROP TABLE @returnlist_1` internally where makes logicalschema = NULL.